torch/_inductor/graph.py

```diff
@@ -72,7 +72,17 @@ def symbolic_sizes_strides(self, ex: torch.Tensor):
                 ex.stride()
             )
         else:
-            size, stride = self._shape_env.create_symbolic_sizes_strides(ex)
+            from torch._dynamo.source import ConstantSource
+
+            # TODO: make a dedicated UnknownSource for this?
+            source = ConstantSource(
+                f"__unknown_tensor_{len(self._shape_env.var_to_val)}"
+            )
+            (
+                size,
+                stride,
+                _,
+            ) = self._shape_env.create_symbolic_sizes_strides_storage_offset(ex, source)
```

Comment:
`create_symbolic_sizes_strides` no longer exists, was replaced by this.

What's the proper way to create a source here? I've pretty much copy-pasted from `create_symint` in `/test/test_dynamic_shapes.py`.

Reply:
@Chillee and I discussed this and this code should become unreachable when https://github.com/pytorch/pytorch/pull/93059 lands. So I'd also look positively on you turning this into an assert False, or the hack is fine as well and just need to TODO it to be removed with @Chillee's patch.